### ISSUE TITLE
Peer Review

### ISSUE THREAD

**Issue by alexandersunliang** [aryamansuri/NBA-3PointAnalysis#2]
Hi! Really interesting topic, the paper is not finished yet but I will comment on what is done

- Beyond the arc is a really cool title
- Abstract is complete, but I would amend with the model results after it is finished
- If there aren't that many "other pertinent variables", it might be beneficial to include them in the abstract in case they have a significant impact on the regression
- Although no placeholders yet, once the paper is finished it would be nice to have hyperlinks to the page numbers for each section in the introduction
- Data section is complete
- Data cleaning process is included
- **No R citation, only the autocomplete one within model**
- Model justification seems semi-complete, no actual justification of the model chosen seems to be provided
- Discussion is semi-complete
- Although the Rockets lived and died by the three, didn't the KD warriors also heavily rely on the three-point shot albeit not as much?

Really exciting paper, I look forward to reading the rest!
